tracetools_launch/tracetools_launch/action.py

```diff
@@ -445,7 +473,12 @@ def setup() -> bool:
                 )
                 if self._trace_directory is None:
                     return False
-                self._logger.info(f'Writing tracing session to: {self._trace_directory}')
+                if not dual_session:
+                    self._logger.info(f'Writing tracing session to: {self._trace_directory}')
+                else:
+                    self._logger.info(
+                        'Snapshot tracing session will be written to: '
+                        f'{self._trace_directory}')
```

Comment:
Follow-up to https://github.com/ros2/ros2_tracing/pull/191#discussion_r2289195467 regarding the above change.

This is similar to passive vs active voice. It's usually better to use active voice for technical documentation, befcause it's simpler. See:

1. From a general writing perspective: https://www.grammarly.com/blog/sentences/active-vs-passive-voice/
2. From a technical documentation writing perspective: https://developers.google.com/tech-writing/one/active-voice

Active voice is closer to "_Writing_ snapshot tracing session to" while passive voice is closer to "Snapshot tracing session _will be written_ to"

```suggestion
                    self._logger.info(
                        f'Writing snapshot tracing session to: {self._trace_directory}')
```

Reply:
as discussed offline, we can just remove the condition and keep the original log